pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test: TestSQLStatsRegions failed

pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test.TestSQLStatsRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/2e2835b9-8c46-456e-a322-9b523e2f551c?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3Fsc3RhdHNjY2w6c3Fsc3RhdHNjY2xfdGVzdA==) on master @ [d0582983521533ffc15a8b8525a188d80da2a0b7](https://github.com/cockroachdb/cockroach/commits/d0582983521533ffc15a8b8525a188d80da2a0b7):


```
=== RUN   TestSQLStatsRegions
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestSQLStatsRegions3100088018
    test_log_scope.go:81: use -show-logs to present logs inline
    sql_stats_test.go:246: -- test log scope end --
test logs left over in: outputs.zip/logTestSQLStatsRegions3100088018
--- FAIL: TestSQLStatsRegions (156.94s)
=== RUN   TestSQLStatsRegions/system_tenant
    sql_stats_test.go:241: 
        	Error Trace:	pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test_test/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go:241
        	            				github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            				github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            				github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            				github.com/cockroachdb/cockroach/pkg/testutils/soon.go:60
        	            				pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test_test/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go:188
        	Error:      	Not equal: 
        	            	expected: []string{"gcp-us-central1", "gcp-us-east1", "gcp-us-west1"}
        	            	actual  : []string{"gcp-us-central1", "gcp-us-west1"}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,4 +1,3 @@
        	            	-([]string) (len=3) {
        	            	+([]string) (len=2) {
        	            	  (string) (len=15) "gcp-us-central1",
        	            	- (string) (len=12) "gcp-us-east1",
        	            	  (string) (len=12) "gcp-us-west1"
        	Test:       	TestSQLStatsRegions/system_tenant
    --- FAIL: TestSQLStatsRegions/system_tenant (110.45s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSQLStatsRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39311

--- a/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go
+++ b/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go
@@ -37,8 +37,7 @@ func TestSQLStatsRegions(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.UnderRace(t, "test is to slow for race")
-	skip.UnderStress(t, "test is too heavy to run under stress")
+	skip.UnderDuress(t, "test is too heavy for special configs")
 
 	// We build a small multiregion cluster, with the proper settings for
 	// multi-region tenants, then run tests over both the system tenant

